Nos/Models/FlagOption.swift

```diff
@@ -8,62 +8,82 @@ struct FlagOption: Identifiable, Equatable {
     /// An optional description that provides more detail about the flagging option.
     let description: String?
 
-    /// An optional message that will be displayed when the user has selected a particular flag.
-    let info: String?
+    /// An optional closure that returns a message to display when the user selects a particular flag.
+    /// The closure takes the title of the previously selected flag as an optional `String` 
+    /// and returns an optional message to provide additional context or details for the current flag.
+    /// - Parameter String?: The title of the previously selected flag, if available.
+    /// - Returns: A message related to the current flag based on the previous selection, 
+    /// or `nil` if no message is provided.
+    let info: ((String?) -> String?)?
```

Comment:
I wish it could be clear that the previously selected flag will be inserted into the localizable string for the info box, but I'm not sure how to do that. It's not a big deal, though.